docs/tls/CONFIGURATION.md

```diff
@@ -0,0 +1,164 @@
+  tls.key: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2Z0lCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktnd2dnU2tBZ0VBQW9JQkFRREVWYlFLc0VaQjlpNnkKOWhjM0c5dXhxZUkwTmI0R25TYWdLUElQZk1BMUJ5MENLNzErRHQzbmhDRWNwTXRabjVGM2JzdGhpU0ZOa3VsNwpDdjVpMkYwM3R6eGhRbDVjcEdWWENJTGpmTjJ6OE4wbWNvY01LNWVoYnBVYlB5cFdHZFAwZmZiWktYT2VkSlB0Ck1iUkh3T1haaWlheCthZFU3SGlaQ00zZXRxaEZnZ0x4ajFiVzZRTkI4UEIrVTNxQmJERExMa0E2MkF3UTBtQkcKbGJvOG1RWVdjS1dYaTA2RnZqYmRQZHRXZDZEbmR0SzdEY0daYzllM092SW0wQ3l2amNTRkxaVHlkYWNRb2YzMQpGU1hGTnF0ekh6KzZlTFZMenpBODdBTUwxeE00ck1kcUhCTExBdnFFUlZyVmRjcHcxUlZ5ZmpPdTVLMVFTMCtECkpKUUtpc0Y1QWdNQkFBRUNnZ0VBUmJEaVFyVUZpMTUyZnlIcFphOW0wVzZyRTd0TDQ0OGU2ZFQrNzA1cDc2akgKYWpEYWg0ZUJieSt3alRsVjBVOVoxRm9vcVV1QzNjVmJTVWpDdVpWeHFWaEtmMytPZlRZQm50NHZqODUvNStLRQpyYVFUbWRkVGNJTVBQR2UwZ0hHVWtnR3dnSzg5RWk3eFgwMjRYNHZ5U3NER01ZQ2J6QUpRWUlvR2dacTJxa3ozCnNxOTA2WmM0dC9EaHVQK1lXYUN0R2xYUFc5OHlMb20wWVNQWmlncXN3OGV0ZDQvMVFHVFpYKzh4QkdSNXpwYXoKa0g5MkhrWXFUWlgyNnpoMnpGSzIvS2ZtRy94OUFCeTlUaHo0bjZDV21wd21ZV2pFbWoxbkVPS2dHd1BaYWRadwphcDd3SEhCRWhrVjZyZkgyQnM2Ym5QTHpUN0dnMDZTZk1WWnBMbHBGalFLQmdRRHBYNnR3Q2tCUGRGVzR5Y1JLCmRWeDZQRnRwZzVZcG1aS1FNN2FBSlpIK2ZLRExvSTRVRDMxQUFQQS9OVGw5OHdnQngwYTZ3SXlGTTljMnViQkwKa09HNTlmRTQrR3RpTjBSNVA2OHZrZDcvblJSQ3hQenRXZ3F0VHpCS3JRMC9qRjdHcnl1UjZxZVp4MXpKQUNQcwpxazJtMEorQ3Vxd2piNHdhU2dYQTVlWGhwd0tCZ1FEWFhycG9XeWYweGNPeU8xbm1TclhVbklEVitKWG5kdjlXCmRxNWhpdW9mYTdoRURyaXV5YVZBYzJFTFp1VjdOZ1BEQkt0U1JwVlU2anhSc1Nsb3ZpckVUVlNhNFZtb0FVS0UKL3c5ZWtwVys0Z1RGUjg2cTVHV2dYTjlENFNrT1Vlendna0E1USt3aC9xa3gyTFBBcGgwT25HMXJBejRuL1RCWgpGbXFRVUpkbjN3S0JnRXJmV0w5UGFxYVRLbFZvMWtNVk9DQTNjUlIvRlJRdm95Z0VqMDY1dndjb3NPWGprdkc0CnJuUjBXSi9PMGEwbzVhNjhZbUxVRjhFOGlwV1NLWTkwNGhhcFkwVGwzbjNhUVd1dEZPVnNKV0ZZNGdFeDlmdW8KWWJpTXFXK3I2WTR6Mk14aHN0YlR0QVhRUW1KeVd4UTQ1MFRYdGZWWk1wVUQycTBzaG9BR3dpUU5Bb0dCQUo1eQpJNlBZeEhtUThWdHp6THpmWkF5S0kwTElBc2pxUkFvQmdHTmt4ci9kQ1JDUTRCQS9hNk1SLzJoYkNDTndacTRZCkJrNmVmUWllMTBlbWxmZG9TS0JvNEFDZUg4NjVObzRQOCtlNDRoQXZ5aUMxcG5jVlZ5cjJpSTh3M0hxYmpUeVkKSnRXNmdjTzB6aGRNL1ZtVEhEZCtvQVVsdXBuQzdNbnQ4dkkvU0ZSekFvR0JBSmxRMlBHLzdsUUZydXF6VU1mTApvT0o4NDRXR2d3cUF0YVM0SmZFdGpOWU8zMjdhaVdsRHpzRElYczVvaVk1NUtHaTZaZDVpaHlGMEJGQlRkR1I5CklKd3l5b0N3QktVNnBwTEZBdjI0cGdsVzhsM2RUQlFsWlFGUFVqdm40U0NsNnVob0RuMGRCaFVNWVhhcFNxUWsKb2Q3SzErYWNmeThVS200TWRuR0FBa2dTCi0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K
+kind: Secret
+metadata:
+  creationTimestamp: null
```

Comment:
given the metadata - is this a describe?  Or an example of how to define one?
Given the audience guidance comment, it should be guidance on how to define one.

Reply:
Yes, this is how to define the Secret. The `creationTimestamp` is an artifact generated by the `--dry-run=client -o yaml` flags. Will remove it as it is somewhat distracting.